packages/zmarkdown/common.js

```diff
@@ -127,34 +120,41 @@ const zmdParser = (config, target) => {
         vfile.fail(`Markdown AST too complex: tree depth > ${config.maxNesting}`)
       }
     })
-}
 
-function getLatexProcessor (remarkConfig, rebberConfig, target) {
-  remarkConfig.noTypography = true
+  if (extraRemarkPlugins && extraRemarkPlugins.length > 0) {
```

Comment:
I think you could remove this condition if you do `const zmdParser = (config, extraRemarkPlugins = []) => {` on l.70.